cypress/support/step_definitions/dashboard/i_am_logged_in_as_a_string.js

```diff
@@ -3,18 +3,18 @@ Given("I am logged in as a {string}", (role) => {
     operation: "user.upsert",
     variables: {
       create: {
-        email: "smoke@cautionyourblast.com",
+        email: "ali+test@cautionyourblast.com",
         jsonData: { roles: [role] },
       },
       update: {
         jsonData: { roles: [role] },
       },
       where: {
-        email: "smoke@cautionyourblast.com",
+        email: "ali+test@cautionyourblast.com",
```

Comment:
revert please, was deliberately `smoke@cautionyourblast.com`

Reply:
This should have been reverted prior to committing.  Your change to include isTest to isCybDev should do the job to redirect when in CI.  Will also need a similar condition when sending the emails for publishing and requesting edits - I'll sort this out.